Berksfile

```diff
@@ -1,3 +1,8 @@
 source 'https://supermarket.chef.io'
 
 metadata
+
+group :solo do
```

Comment:
I am not sure exactly, I just saw it in other cookbooks like this so I thought it was how it was supposed to be done.